katas/content/multi_qubit_measurements/index.md

```diff
@@ -284,7 +284,7 @@ The action of $P_0$ on $\ket \psi$ is
 
 $$P_0 \ket \psi = \left(\ket 0\bra 0 \otimes \mathbb{1}\right) \frac{1}{\sqrt 2}\big(\ket{01} - \ket{10}\big) =$$
 
-$$= \frac{1}{\sqrt 2} \big( \ket 0\bra 0 0\rangle \otimes \mathbb{1} \ket{1} - \ket 0 \bra 0 1\rangle \otimes \mathbb{1} \ket 0 \big) = \frac{1}{\sqrt 2} \ket{01}$$
+$$= \frac{1}{\sqrt 2} \big( \ket 0 \braket{0|0} \otimes \mathbb{1} \ket{1} - \ket 0 \braket{0|1} \otimes \mathbb{1} \ket 0 \big) = \frac{1}{\sqrt 2} \ket{01}$$
```

Comment:
Ooh, nice catch. It's these kind of cases that make it not just a simple find / replace!

Reply:
Yes, I have _feelings_ about \ket and \bra without squiggly brackets after them too, but I didn't want to pile those changes on top of the ones I was already making. Some other day :-)